static/app/views/issueList/overview.tsx

```diff
@@ -263,8 +260,10 @@ class IssueListOverview extends Component<Props, State> {
       prevUrlQuery.cursor !== newUrlQuery.cursor ||
       prevUrlQuery.statsPeriod !== newUrlQuery.statsPeriod ||
       prevUrlQuery.groupStatsPeriod !== newUrlQuery.groupStatsPeriod ||
-      prevQuery !== newQuery ||
-      prevSort !== newSort
+      prevSort !== newSort ||
+      // Ignore query changes when issue-stream-performance is live
+      (!this.props.organization.features.includes('issue-stream-performance') &&
+        prevQuery !== newQuery)
```

Comment:
will have to change this, breaks the query updating when navigating using forward/back